### PR TITLE
Update submission_clusters.js.coffee and show.json.erb to display per…

### PR DESCRIPTION
**Pull request by nguyen-mai-huong**
adding similarity percentages to Similarity Cluster graphs

### CODE REVIEW

File: app/assets/javascripts/submission_clusters.js.coffee [hunk 1]
```diff
@@ -72,7 +72,7 @@ SubmissionCluster.drawCluster = (clusterId, data) ->
           left = 0.5 * (parseFloat(nodeFromLeft) + parseFloat(nodeToLeft)) - 5
           top = 0.5 * (parseFloat(nodeFromTop) + parseFloat(nodeToTop)) - 8
           style = "position: absolute; left: "+left+"px; top: "+top+"px;"
-          $("#canvas_wrapper_"+clusterId+"-label").append("<div class=\"edgeLabel\" style=\""+style+"\"><a href=\""+adj.data.submissionSimilarityUrl+"\">view</div>")
+          $("#canvas_wrapper_"+clusterId+"-label").append("<div class=\"edgeLabel\" style=\""+style+"\"><a href=\""+adj.data.submissionSimilarityUrl+"\" style=\"margin-right: 10px\">view</a><span>\""+adj.data.submissionSimilarityPercentage+"\"</span></div>")
```
Comment (knmnyn) [inline app/assets/javascripts/submission_clusters.js.coffee:75]: UI string could be better, but ok.

**Review by knmnyn — APPROVED** [review]

**Comment by knmnyn** [comment]
PR could be more detailed to add in a screenshot of the (intended) outcome.

**Review by Riyas97 — CHANGES_REQUESTED** [review]
Code LGTM. But, I just have a small request. Can you post a pic about how the UI would look like? In the past, when I was working on similar issues, I had to change the css file. So, I just wanted to check whether such changes would be necessary here. Thanks a lot.

**Comment by nguyen-mai-huong** [comment]
> Code LGTM. But, I just have a small request. Can you post a pic about how the UI would look like? In the past, when I was working on similar issues, I had to change the css file. So, I just wanted to check whether such changes would be necessary here. Thanks a lot.

Hi Riyas, attached is how UI looks like.

graph-with-percentage

**Review by Riyas97 — APPROVED** [review]

**Comment by Riyas97** [comment]
Thanks.

**Pull request merged**